Parsers/TrendMicro/TrendMicroTippingPoint

```diff
@@ -0,0 +1,28 @@
+// Title:           Trend Micro TippingPoint Data Parser
+// Author:          Trend Micro
+// Version:         1.0
+// Last Updated:    11/05/2020
+// Comment:         Initial Release
+//  
+// DESCRIPTION:
+// This parser takes the TippingPoint SMS ArcSight CEF Format v4.2 logs from a Syslog data stream and parses the data into a normalized schema
+//
+// USAGE:
+// 1. Open Log Analytics/Azure Sentinel Logs blade. Copy the query below and paste into the Logs query window. 
+// 2. Run the query to ensure it's correctly parsing the data being sent to Azure Sentinel in CEF format.  
+//    The presence of new fields such as TippingPointVLAN indicate the function should be working as expected.
+// 3. Click the Save button above the query. A pane will appear on the right, select "as Function" from the drop down. Enter a Function Name.
+//    In order for the Trend Micro TippingPoint data connector, and future enhancements to work correctly, the Function Alias must be set to - TrendMicroTippingPoint.
+// 4. Function App usually take 10-15 minutes to activate. You can then use the Function Alias for other queries.
+//
+// REFERENCES: 
+// Using functions in Azure monitor log queries: https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+// TippingPoint SMS User Guide: https://docs.trendmicro.com/all/tip/sms/v5.4.0/en-us/sms_5.4.0_ug.pdf
+// 
+CommonSecurityLog
+| where DeviceProduct == "UnityOne"
+| extend DeviceVendor = iff(DeviceVendor == "TippingPoint", "Trend Micro", DeviceVendor)
+| extend DeviceProduct = iff(DeviceProduct == "UnityOne", "TippingPoint", DeviceProduct)
+| parse AdditionalExtensions with "cat=" cat
+| project-rename TippingPointVLAN = DeviceCustomNumber1, TippingPointTaxonomy = DeviceCustomNumber2, TippingPointPacketTrace = DeviceCustomNumber3, TippingPointProfileName = DeviceCustomString1, TippingPointPolicyUUID = DeviceCustomString2, TippingPointSignatureUUID = DeviceCustomString3, TippingPointZoneNames = DeviceCustomString4, TippingPointSMSName = DeviceCustomString5, TippingPointFilterMessageParms = DeviceCustomString6
+| project-away DeviceCustomIPv6Address1Label, DeviceCustomIPv6Address2Label, DeviceCustomIPv6Address3Label, DeviceCustomNumber1Label, DeviceCustomNumber2Label, DeviceCustomNumber3Label, DeviceCustomString1Label, DeviceCustomString2Label, DeviceCustomString3Label, DeviceCustomString4Label, DeviceCustomString5Label, DeviceCustomString6Label
```

Comment:
@mikedgibson, according to Offir, the project-away clause is not needed for rename columns, do you try it and believe project-away is still needed for the parser?